src/pynwb/core.py

```diff
@@ -332,309 +316,6 @@ def __getitem__(self, idx):
         return self.__regionslicer[idx]
 
 
-class MultiContainerInterface(NWBDataInterface):
-    '''
-    A class for dynamically defining a API classes that
-    represent NWBDataInterfaces that contain multiple Containers
-    of the same type
-
-    To use, extend this class, and create a dictionary as a class
-    attribute with the following keys:
-
-    * 'add' to name the method for adding Container instances
-
-    * 'create' to name the method fo creating Container instances
-
-    * 'get' to name the method for getting Container instances
-
-    * 'attr' to name the attribute that stores the Container instances
-
-    * 'type' to provide the Container object type
-
-    See LFP or Position for an example of how to use this.
-    '''
+class MultiContainerInterface(NWBDataInterface, hdmf_MultiContainerInterface):
```

Comment:
Add docstring to the at least give a hint why this class is here e.g,: "Defined in PyNWB for API backward compatibility. See HDMF MultiContainterInterface for details".

Reply:
```suggestion
class MultiContainerInterface(NWBDataInterface, hdmf_MultiContainerInterface):
    """Defined in PyNWB for API backward compatibility. See HDMF MultiContainterInterface for details"""
```